docs/content/docs.md

```diff
@@ -3,59 +3,14 @@ title: Porter Docs
 description: All the magic of Porter explained
 ---
 
-Porter takes the work out of creating CNAB bundles. It provides a declarative authoring 
-experience that lets you to reuse existing bundles, and understands how to translate 
-CNAB actions to Helm, Terraform, Azure, etc.
+Porter is an open source project that lets you package your application artifact, client tools, configuration and deployment logic together as a versioned bundle that you can distribute, and then install with a single command.
 
-```console
-$ porter create
-created porter.yaml
+## Explore documentation
 
-$ porter build
-created Dockerfile
-created bundle.json
-created cnab/app/run
+Porter has a lot of documentation and we're in the process of reorganizing and updating it. We welcome [contributions](/contribute/) from the community. 
 
-$ porter install
-```
+## Quicklinks
```

Comment:
Do you mind if we continue calling this section "Next Steps"? My goal was to have a section like this at the bottom of pages that lead readers to good things to read next. Quicklinks doesn't quite imply the same thing.

Reply:
What about having both? As the landing page I was thinking this is like Step 0 so here are a set of areas you could go to versus "Next"

Reply:
Ah yeah I see how the main docs page is difference. Makes sense. 👍